packages/tools/client-debugger/client-debugger/src/FluidDebuggerLogger.ts

```diff
@@ -21,6 +30,52 @@ import { debuggerMessageSource, postMessagesToWindow, TelemetryEventMessage } fr
  * @internal
  */
 export class FluidDebuggerLogger extends TelemetryLogger {
```

Comment:
Can we document the Window messages we listen for and post in the type documentation here? See `FluidClientDebugger` for an example of this.

Reply:
Yes. Updating doc here...

Reply:
Thanks!